wp-content/themes/csisnuclearnetwork/assets/_scss/components/_post-block-related.scss

```diff
@@ -38,6 +53,6 @@
     margin-top: rem(16);
     margin-bottom: rem(4);
     @extend %text-style-label-medium;
-    font-weight: 400;
+    color: $color-black-165;
```

Comment:
The hover and focus styles can be found [here](https://www.figma.com/file/cAg0A1ww0s2KbJdIbUvl2j/PONI-Redesign-Handoff?node-id=157%3A86903).